src/components/DescriptionList/_description-list.scss

```diff
@@ -9,5 +9,9 @@
 
 .cc-description-list__definition {
   display: block;
-  margin: 0 0 var(--space-sm) 0;
+  margin: 0 0 var(--space-md) 0;
```

Comment:
The issue doesn't mention the margins, so just checking this change was intentional?

Reply:
Hi .. yes, this is intentional.